client/src/lib/bowling.js

```diff
@@ -0,0 +1,71 @@
+class Bowling {
```

Comment:
I want to query this name as being a little vague. A class called Bowling might represent the game, the bowling alley, or something else.